elastic/kibana
# [Fleet] Fix changing space for agent actions

## Summary

Resolve https://github.com/elastic/kibana/issues/194931

When reassigning a policy to different spaces we were not reassigning action that PR fix it.

## How to test

Enable space awareness (feature flag + API call)
Create agent action in a space and update the policy spaces, it should update the action spaces, too and action should be visible in agent activity of the newly assigned space.

<img width="1508" alt="Screenshot 2024-12-10 at 11 34 23 AM" src="https://github.com/user-attachments/assets/6e7a7fb2-5601-4732-97ca-75c8a0382ad3">
